client/src/modules/authorization/AuthBox.jsx

```diff
@@ -147,11 +180,11 @@ class AuthBox extends PureComponent {
             <div className="authbox__intro">
               <AppLogo />
             </div>
-            {isSignUpFormVisible ? (
-              <SignUpForm onCancel={this.handleSignUpFormVisibility} />
+            {isSignUpFormVisible || isSignInFormVisible ? (
```

Comment:
you are checking it here, and inside renderForms method.
Does it make sense to check it 2 times?

wouldn't be enough  to just
`this.renderForms()` ?

Reply:
Not exactly, because there is a ternary operator if signIn- or signUp- form isn't displayed buttons are displayed. 
So for the first time, I check if any of these forms should be displayed and inside this method, I have to check which one should be displayed.
I can leave this like it is
or change it to 3 separate conditions:
```
if(isSignUpFormVisible && (...)}
if(isSignInFormVisible && (...)}
if(!isSignInFormVisible &&  !isSignUpFormVisible && (...)}
```
for now, I change it to (it it almost the same but maybe more clear:
```
const areFormsVisible = isSignUpFormVisible || isSignInFormVisible;
(...)
{areFormsVisible ? (
  this.renderForms()
) : (
  <Fragment>
    {this.renderSignInButtons()}
    {this.renderSignUpButton()}
     {this.renderDemoButton()}
   </Fragment>
 )}
(...)
}

Reply:
> const areFormsVisible = isSignUpFormVisible || isSignInFormVisible;
> (...)
> {areFormsVisible ? (
>   this.renderForms()
> ) : (
>   <Fragment>
>     {this.renderSignInButtons()}
>     {this.renderSignUpButton()}
>      {this.renderDemoButton()}
>    </Fragment>
>  )}
> (...)
> }

Leave like this.